### ISSUE TITLE
Layer style overrides treated as unused styles

### ISSUE THREAD

**Issue by avkvirtru** [sonburn/unused-style-remover#3]
Hey @sonburn, thanks for another handy plugin.

I noticed that if I use a layer style only as an override (and not anywhere else in my Sketch file), it is treated as an unused style. Is this intentional? Is there any way to detect styles as overrides and treat them as used? I ask on behalf of an emerging and increasingly complicated design system.

**Comment by jordanborth** [comment]
I'm also running in to this issue

**Comment by rafaelmatosdasilva** [comment]
yup, having the same issue which unfortunately makes this plugin unusable.

**Comment by mariuszmickiewicz** [comment]
I'm not sure, it might be related. I have a Layer Style which is "transparent", I use it to override (turn off) some Layer Styles inside the Symbols and it is listed as "unused" style. Not sure if because it is only used as override or because it doesn't have a fill or a border.

**Comment by janospower** [comment]
Same issue here, a fix would be greatly appreciated!

**Comment by appscaptain** [comment]
Same here. No plans for fixing it? It really makes it unuseable.
